woocommerce/woocommerce
# Fix system status connected to woocommerce.com status

### All Submissions:






### Changes proposed in this Pull Request:
This PR fixes a logic error in the system status page where it was not checking for the value yes but instead just checking that the variable was not empty resulting in it always showing that you are connected to woocommerce.com even though you are not.


Closes #23606 

### How to test the changes in this Pull Request:

1. Disconnect from woo.com and check system status, it should not have a checkmark next to connected to woocommerce.com option.
2. Connect to woo.com and check system status again, the option should now have a checkmark.

### Other information:




### Changelog entry

> Fix - Always showing connected to woocommerce.com in system status report.
